scripts/run_daq_utils.py

```diff
@@ -0,0 +1,27 @@
+import os,sys,time
+from daq_utils import DaqManager
+import getopt
+
+help_messages = {
+        'wheredaq': 'Discover what host is running the daq in the current hutch, if any.',
+        'stopdaq': 'Stop the daq in the current hutch.',
+        'restartdaq': 'Verify requirements for running the daq then stop and start it.',
+        'isdaqbatch': "Determine if the current user's hutch uses daqbatch",
+        }
+
+if __name__ == "__main__":
+    commands = ['wheredaq', 'stopdaq', 'restartdaq', 'isdaqbatch']
+    opts, args = getopt.getopt(sys.argv[2:], 'hm:dv', ["help","dss"])
```

Comment:
Consider defaulting to `argparse` instead of `getopt` to get free consistent help text and more readable CLI input parsing.

Reply:
Upvote for argparse. In order to get this rolled out though, could we create an issue to solve in a future and hopefully near update @ZLLentz ?

Reply:
@silkenelson we can definitely ignore any of the comments per our discretion and promise to do them later, but I caution against incorporating code in our repos that we will have friction maintaining later. It's better to do it cleanly the first time.

If we are to become maintainers of this (as implied by a PR into engineering tools) then we should be at least a little bit picky. If the daq team is going to be the maintainer of this then this PR shouldn't have so much code in it, these scripts should live elsewhere and only be referenced here (or otherwise included in our startup scripts)

I understand of course that this PR is necessary to continue smooth operation of the DAQ given the context. So if you think it's OK feel free to approve and merge.

Reply:
Fixed. The use of argparse is added in the main file. This also resolves the issues with arguments being parsed at different places in the code.